app/queue/notifications.py

```diff
@@ -92,6 +92,32 @@ class SystemDeletedSchema(NotificationSchema):
     events = fields.List(fields.Nested(SystemDeletedEventListSchema()))
 
 
+# System became stale notification
+class SystemStaleContextSchema(MarshmallowSchema):
+    inventory_id = fields.Str(required=True)
+    hostname = fields.Str(required=True)
```

Comment:
OK, then I'll suggest:
```suggestion
    fqdn = fields.Str(required=True)
```